sandbox/index.html.example

```diff
@@ -4,19 +4,25 @@
   <meta charset="utf-8" />
   <title>Video.js Sandbox</title>
 
-  <link href="../build/files/video-js.css" rel="stylesheet" type="text/css">
+  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
 
   <!-- LOAD VIDEO.JS SOURCE FILES IN ORDER -->
-  <script src="../build/source-loader.js"></script>
+  <script src="../build/temp/video.js"></script>
```

Comment:
I use watchify and karma's watch to have continuous tests running in [videojs-playlist](https://github.com/brightcove/videojs-playlist/blob/master/package.json#L6-L17)